tests/unit/Infrastructure/DashboardBundle/Factory/EntityTypeFactoryTest.php

```diff
@@ -225,4 +226,140 @@ public function test_build_create_new_oauth20_rs_form()
 
         $this->assertInstanceOf(FormType::class, $form);
     }
+
+    /**
+     *  @dataProvider testOptions
```

Comment:
Nitpick

Please leave as is, but next time try to use a somewhat more descriptive name, like: `providePublishOrChangeOptions` your provider also describes what it does, but imagine this class having multiple tests with similarly named providers. Then it would soon be hard to distinguish between them.